packages/polar/test/helpers/project.ts

```diff
@@ -0,0 +1,81 @@
+import * as fs from "fs";
+import * as fsExtra from "fs-extra";
+import path from "path";
+
+import { TASK_CLEAN } from "../../src/builtin-tasks/task-names";
+import { PolarRuntimeEnvironment } from "../../src/types";
+import { useEnvironment } from "./environment";
+
+export const testFixtureOutputFile = "output.txt";
```

Comment:
This can go in project structure.

Reply:
this is part of test folder, it won't be used in src.

Reply:
we can have tests constants file common for all tests?